src/less/blog-v2/includes/footer.less

```diff
@@ -86,28 +86,33 @@ footer {
     }
 
     .-icon-facebook {
-      background-image: url('https://images.ctfassets.net/awpxl2koull4/2X6c8ywqRoOjil1030aGjt/40f2ff9c731c7dc90f7a64392ce29374/social-footer-icons.png');
+      background-image: url('https://images.ctfassets.net/awpxl2koull4/2X6c8ywqRoOjil1030aGjt/b8659a27ccfa58311a6bf582c66e9fc2/social-footer-icons.png');
       background-position: 0 0;
       background-repeat: no-repeat;
     }
 
     .-icon-instagram {
-      background-image: url('https://images.ctfassets.net/awpxl2koull4/2X6c8ywqRoOjil1030aGjt/40f2ff9c731c7dc90f7a64392ce29374/social-footer-icons.png');
+      background-image: url('https://images.ctfassets.net/awpxl2koull4/2X6c8ywqRoOjil1030aGjt/b8659a27ccfa58311a6bf582c66e9fc2/social-footer-icons.png');
       background-position: -50px 0;
       background-repeat: no-repeat;
     }
 
     .-icon-twitter {
-      background-image: url('https://images.ctfassets.net/awpxl2koull4/2X6c8ywqRoOjil1030aGjt/40f2ff9c731c7dc90f7a64392ce29374/social-footer-icons.png');
+      background-image: url('https://images.ctfassets.net/awpxl2koull4/2X6c8ywqRoOjil1030aGjt/b8659a27ccfa58311a6bf582c66e9fc2/social-footer-icons.png');
       background-position: -100px 0;
       background-repeat: no-repeat;
     }
 
     .-icon-pinterest {
-      background-image: url('https://images.ctfassets.net/awpxl2koull4/2X6c8ywqRoOjil1030aGjt/40f2ff9c731c7dc90f7a64392ce29374/social-footer-icons.png');
+      background-image: url('https://images.ctfassets.net/awpxl2koull4/2X6c8ywqRoOjil1030aGjt/b8659a27ccfa58311a6bf582c66e9fc2/social-footer-icons.png');
       background-position: -150px 0;
       background-repeat: no-repeat;
     }
+    .-icon-spotify {
+      background-image: url('https://images.ctfassets.net/awpxl2koull4/2X6c8ywqRoOjil1030aGjt/b8659a27ccfa58311a6bf582c66e9fc2/social-footer-icons.png');
+      background-position: -201px 0;
```

Comment:
Let me know if this feels like too much, but would it be possible to update the asset so that this works with `background-position: -200px 0`? The `201` just makes it seem like it could be a typo/bug to someone new coming into this code.

Reply:
Updated the asset!